packages/sources/cfbenchmarks-test/src/endpoint/common/crypto.ts

```diff
@@ -0,0 +1,127 @@
+import { InputParameters } from '@chainlink/external-adapter-framework/validation/input-params'
+import {
+  PriceEndpoint,
+  priceEndpointInputParameters,
+} from '@chainlink/external-adapter-framework/adapter'
+import { RoutingTransport } from '@chainlink/external-adapter-framework/transports/routing'
+import { EmptyObject } from '@chainlink/external-adapter-framework/util'
+import { AdapterInputError } from '@chainlink/external-adapter-framework/validation/error'
+import { transport as rest, transportSecondary as restSecondary } from '../rest/crypto'
+import {
+  transport as websocket,
+  transportSecondary as websocketSecondary,
+} from '../websocket/crypto'
+import { customSettings } from '../../config'
+
+export type Params = { index?: string; base?: string; quote?: string }
+
+const extendedInputParameters: InputParameters = {
+  index: {
+    description: 'The ID of the index. Takes priority over base/quote when provided.',
+    type: 'string',
+    required: false,
+  },
+  base: {
+    aliases: ['from', 'coin'],
+    type: 'string',
+    description: 'The symbol of symbols of the currency to query',
+    required: false,
+  },
+  quote: {
+    aliases: ['to', 'market'],
+    type: 'string',
+    description: 'The symbol of the currency to convert to',
+    required: false,
+  },
+}
+
+export const inputParameters = {
+  ...priceEndpointInputParameters,
+  ...extendedInputParameters,
+}
```

Comment:
Right now the obj merging is not doing much, imo we can simplify like this:

```suggestion
const inputParameters: InputParameters = {
  index: {
    description: 'The ID of the index. Takes priority over base/quote when provided.',
    type: 'string',
    required: false,
  },
  base: {
    ...priceEndpointInputParameters.base,
    required: false
  },
  quote: {
    ...priceEndpointInputParameters.quote,
    required: false
  },
}
```

Reply:
Oop, good catch. Left over from before I realized CFBenchmarks has optional base/quote. Removed.